common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -149,11 +149,8 @@ div.problem {
     }
   }
 
-  span > label {
-    display: block;
-    margin-bottom: $baseline;
-    font: inherit;
-    color: inherit;
+  .question-description {
+    @include margin(($baseline*0.75), 0, ($baseline*0.75), 0);
```

Comment:
two if the margins are same. we should be able to just write `margin(($baseline*0.75), 0);`